curriculum/utils.test.ts

```diff
@@ -31,14 +31,14 @@ describe('getSuperOrder', () => {
     expect(getSuperOrder('javascript-algorithms-and-data-structures')).toBe(1);
     expect(getSuperOrder('front-end-development-libraries')).toBe(2);
     expect(getSuperOrder('data-visualization')).toBe(3);
-    expect(getSuperOrder('relational-database')).toBe(4);
-    expect(getSuperOrder('back-end-development-and-apis')).toBe(5);
-    expect(getSuperOrder('quality-assurance')).toBe(6);
-    expect(getSuperOrder('scientific-computing-with-python')).toBe(7);
-    expect(getSuperOrder('data-analysis-with-python')).toBe(8);
-    expect(getSuperOrder('information-security')).toBe(9);
-    expect(getSuperOrder('machine-learning-with-python')).toBe(10);
-    expect(getSuperOrder('coding-interview-prep')).toBe(11);
+    expect(getSuperOrder('back-end-development-and-apis')).toBe(4);
+    expect(getSuperOrder('quality-assurance')).toBe(5);
+    expect(getSuperOrder('scientific-computing-with-python')).toBe(6);
+    expect(getSuperOrder('data-analysis-with-python')).toBe(7);
+    expect(getSuperOrder('information-security')).toBe(8);
+    expect(getSuperOrder('machine-learning-with-python')).toBe(9);
+    expect(getSuperOrder('coding-interview-prep')).toBe(10);
+    expect(getSuperOrder('relational-database')).toBe(12);
```

Comment:
I see this is quite annoying to change.  I'll try to come up with something a bit less irritating.